pkg/api/common/versions.go

```diff
@@ -198,37 +198,99 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.19.0-beta.0":  true,
 }
 
+// AllKubernetesSupportedVersionsOnAzureStack is a whitelist map of all supported Kubernetes version strings on Azure Stack
+// The bool value indicates if creating new clusters with this version is allowed
+var AllKubernetesSupportedVersionsOnAzureStack = map[string]bool{
```

Comment:
Will you be maintaining these version support lists going forward? In practice, this means when the AKS Engine project moves versions forward for public cloud, Azure Stack won't get that support. That seems sane, just want to clarify.

Reply:
Correct